utilities/DumpUnitaryTest/UnitTest1.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Xunit;
+using Quantum.DumpUnitary;
+
+namespace Test
+{
+    public class UnitTest1
```

Comment:
Could we use more descriptive names here? `DumpUnitaryTest` seems a bit repetitive but very accurate.